package.json

```
@@ -1118,56 +1118,94 @@
     "walkthroughs": [
       {
         "id": "confluent-walkthrough",
-        "title": "Connect to Kafka Clusters",
+        "title": "Get Started with Confluent for VS Code",
         "description": "Interact with Apache Kafka® clusters running locally or on Confluent Cloud.",
         "steps": [
           {
-            "id": "connect-local-cluster",
-            "title": "Connect to Kafka",
-            "description": "**Confluent Cloud Clusters** \nLog in to your Confluent account to see your Confluent Cloud resources. \n[Log in](command:confluent.connections.ccloud.signIn)\n[Sign up for free](https://confluent.cloud/signup?utm_source=vscode-ext)\n**Local Clusters** \nIf you have Kafka running locally, it appears automatically in the Resources view. \n[Start Local Resources](command:confluent.docker.startLocalResources)",
+            "id": "connect-conflulent-cloud",
+            "title": "Connect to Confluent Cloud",
+            "description": "Sign in to see your Confluent Cloud resources.\n[Sign in](command:confluent.connections.ccloud.signIn)\n[Sign up for free](https://confluent.cloud/signup?utm_source=vscode-ext)",
             "media": {
               "image": "resources/walkthrough/connect.png",
               "altText": "Connect to Kafka clusters"
             },
             "completionEvents": [
-              "onCommand:command:confluent.connections.ccloud.signIn",
-              "onCommand:confluent.docker.startLocalResources"
+              "onCommand:command:confluent.connections.ccloud.signIn"
```

Looks like this may have been a typo from a previous branch 🤦
```suggestion
              "onCommand:confluent.connections.ccloud.signIn"
```